BUILDING.md

```diff
@@ -318,36 +318,38 @@ If you are updating tests and want to run tests in a single test file
 (e.g. `test/parallel/test-stream2-transform.js`):
 
 ```text
-$ tools/test.py test/parallel/test-stream2-transform.js
+$ python tools/test.py test/parallel/test-stream2-transform.js
```

Comment:
```suggestion
$ python3 tools/test.py test/parallel/test-stream2-transform.js
```

`python` is at best ambiguous, although most platforms that provide `python` map that to Python 2. Some platforms do not create a `python` executable/symlink for Python 3 and just provide `python3`.